src/_guides/language/spec.md

```diff
@@ -9,26 +9,29 @@ For a gentler introduction to Dart, see the
 
 ## Dart 2
 
+### Latest completed specification
+
 Dart 2 has changed the Dart language in many ways,
 some of which are not backward-compatible.
 For details, see the [Dart 2 page](/dart-2).
 
-The formal language specification is not yet up-to-date.
-For information about Dart 2 changes, see the following resources.
+The formal Dart 2 language specification is not yet up-to-date.
+
+### In-progress specification
+
+New language features are typically described using informal language feature specifications.
+These are located inside the dart-lang/language repo:
+  * [accepted](https://github.com/dart-lang/language/tree/master/accepted): Accepted informal proposals
+  * [working](https://github.com/dart-lang/language/tree/master/working): Drafts of potential features
 
-* Feature specifications in the dart-lang/sdk repo:
-  * [docs/language/informal](https://github.com/dart-lang/sdk/tree/master/docs/language/informal)
-* Feature specifications in the dart-lang/language repo:
-  * [accepted](https://github.com/dart-lang/language/tree/master/accepted)
-  * [working](https://github.com/dart-lang/language/tree/master/working)
-* Dart language specification work in progress:
-  * [Source file (LaTeX format)](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex)
-  * [Community-contributed PDF built from the source file](https://github.com/chalin/dart-spec-and-grammar/blob/master/doc/dartLangSpec.pdf)
+The in-progress formal Dart language specification is written using
+the [LaTeX format)](https://github.com/dart-lang/sdk/blob/master/docs/language/dartLangSpec.tex),
```

Comment:
Let's rephrase this slightly (and get rid of the excess paren):

...is written as [a LaTeX file](...)

Reply:
Done